fitsio/fitslib.py

```diff
@@ -2696,12 +2750,16 @@ def _get_tbl_numpy_dtype(self, colnum, include_endianness=True):
                 addstr=''
             else:
                 addstr='>'
-            if npy_type not in ['u1','i1','S']:
+            if npy_type not in ['u1','i1','S','U']:
```

Comment:
Looking at it now, I am wondering about this change.

Reply:
Looking again, we never read unicode bytes directly. The default numpy unicode type is utf32 IIUIC and this can be stored in big or little endian. Straight bytes cannot be. I don't think want to force the unicode representation in the numpy dtype. Resolving this.